src/common/Navigation.js

```diff
@@ -0,0 +1,11 @@
+import Link from "next/link";
+
+function Navigation(props) {
+  return (
+    <nav>
+      <Link href={props.href}>{props.text}</Link>
+    </nav>
+  );
+}
+
+export default Navigation;
```

Comment:
Can you also add a screenshot of what the UI for this looks like? Just one from local host. Add it to the PR description section for screenshots.

Reply:
Since it's not the main nav bar, there is no UI for this component, except I introduce it into other components like the nav bar.

 Though I'm working on the footer and it would need this component, i will take a screenshot of the UI and possibly add this issue ticket under  related issues.